locales/en.json

```diff
@@ -499,6 +499,9 @@
     "global_settings_setting_postfix_compatibility": "Postfix Compatibility",
     "global_settings_setting_postfix_compatibility_help": "Compatibility vs. security tradeoff for the Postfix server. Affects the ciphers (and other security-related aspects)",
     "global_settings_setting_postfix_name": "Postfix (SMTP email server)",
+    "global_settings_setting_trusted_proxies_list": "Trusted proxies list",
+    "global_settings_setting_trusted_proxies_list_help": "",
```

Comment:
Ideally it's better to add "trusted_proxies_list" to the "settings_without_help_key" here : https://github.com/YunoHost/yunohost/blob/dev/maintenance/missing_i18n_keys.py#L217 (it's a bit hackish but that's what we have for now)

On the other hand, maybe we *do* need an explication here ? If I understand correctly, this is the list of the "front" servers when being the back / reverse-proxied server ?